crates/bin/pindexer/src/dex_ex/mod.rs

```diff
@@ -1397,6 +1398,47 @@ impl Component {
 
         Ok(())
     }
+
+    async fn record_transaction(
+        &self,
+        dbtx: &mut PgTransaction<'_>,
+        time: DateTime,
+        height: u64,
+        transaction_id: [u8; 32],
+        transaction: Transaction,
+    ) -> anyhow::Result<()> {
+        sqlx::query(
+            "INSERT INTO dex_ex_transactions (
+                transaction_id,
+                transaction,
+                height,
+                time
+            ) VALUES ($1, $2, $3, $4)",
+        )
+        .bind(transaction_id)
+        .bind(transaction.encode_to_vec())
```

Comment:
Should this not be the `tx_bytes` ?

Reply:
AIUI, the `transaction.encode_to_vec()` _is_ the same value as `tx_bytes`: the Transaction type was created from the raw bytes, confirming that it's a valid tx, then the valid tx gets roundtripped back to bytes via the [encode_to_vec](https://docs.rs/penumbra-sdk-proto/latest/penumbra_sdk_proto/trait.DomainType.html#method.encode_to_vec) method on the domain type.

Reply:
Yeah, this serves as a safeguard to make sure that whatever we put in there is actually parseable as a transaction, which avoids discovering this later in the dex explorer or something.